module-2/XML/activity1-2.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- This document contains business cards and related information -->
+<persons>
+    <person>
+        <name>Rorie Demanche</name>
+        <contact_home ext="+1">7101534949</contact_home>
+        <contact_work ext="+1">5033193558</contact_work>
+        <contact_mobile ext="+1">1827481667</contact_mobile>
```

Comment:
```suggestion
        <contact type="home" ext="+1">7101534949</contact_home>
        <contact type="work" ext="+1">5033193558</contact_work>
        <contact type="mobile" ext="+1">1827481667</contact_mobile>
```
this one would make more sense. better to have it as an attribute, than as a suffix

Reply:
Oh I see. Thank you sir.